index.html

```diff
@@ -635,9 +635,13 @@
 to the people using a system.
 
 In specific cases, [=people=] should be able to [=consent=] to more
-sensitive [=purposes=], such as having their [=identity=] recognised across contexts or
-their reading history shared with a company. The burden of proof on ensuring that informed
-[=consent=] has been obtained needs to be very high in this case.
+sensitive [=processing=], such as having their [=identity=] recognised across contexts or
+their reading history shared with a company. [=Actors=] need to
```

Comment:
See https://github.com/w3ctag/privacy-principles/issues/229#issuecomment-1479981172 for some better wordinng for this sentence.